docs/geedocs/5.2.5/answer/7160006.html

```diff
@@ -114,20 +121,35 @@ <h5>
               Updating the state of asset versions used in large builds (such as when canceling or cleaning a project) is now much more efficient.
             </td>
           </tr>
+          <tr>
+            <td>1037</td>
+            <td>Projects use different level numbers on the command line than in the Fusion User Interface</td>
+            <td>A new parameter, <code>--maxleveloverride</code>, has been added for the <code>genew*project</code>, <code>geaddto*project</code>, and <code>gemodify*project</code> families of command line utilities. This parameter is to be used instead of the <code>--maxlevel</code>, which is now deprecated. The new parameter uses the Terrain and Imagery levels found in the Fusion UI, so they match.</td>
```

Comment:
```suggestion
            <td>A new parameter, <code>--maxleveloverride</code>, has been added for the <code>genew*project</code>, <code>geaddto*project</code>, and <code>gemodify*project</code> families of command line utilities. This parameter should be used instead of <code>--maxlevel</code>, which is now deprecated. The new parameter matches the Terrain and Imagery levels displayed in the Fusion UI</td>
```